content/streamlit-cloud/get-started/manage-your-app.md

```diff
@@ -93,7 +93,9 @@ It is best practice to pin your Streamlit version in `requirements.txt`. Otherw
 
 ### Resource limits
 
-The exact resources and limits will depend on [your workspace plan](https://streamlit.io/cloud). If you need more apps or more resources for your apps you can upgrade your plan or reach out to [support@streamlit.io](mailto:support@streamlit.io).
+<!-- The exact resources and limits will depend on [your workspace plan](https://streamlit.io/cloud). If you need more apps or more resources for your apps you can upgrade your plan or reach out to [support@streamlit.io](mailto:support@streamlit.io). -->
+
+All Community Cloud users have access to the same resources are subjected to the same limits. If you need more apps or more resources for your apps you can reach out to us on our [Community forum](https://discuss.streamlit.io/).
```

Comment:
This sentence is missing a word somewhere. Maybe "and subjected to the same limits"?

Reply:
Thanks! Fixed on [fae1305].(https://github.com/streamlit/docs/pull/387/commits/fae1305bf3ea880a8d0e0fa157bb501d3c8513d8)